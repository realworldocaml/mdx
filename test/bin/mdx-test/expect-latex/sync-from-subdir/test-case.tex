You can sync files from subdirs:

% $MDX file=some_dir/some_module.ml,part=1
\begin{ocaml}
\end{ocaml}
