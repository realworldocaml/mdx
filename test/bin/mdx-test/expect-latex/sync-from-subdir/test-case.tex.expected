You can sync files from subdirs:

% $MDX file=some_dir/some_module.ml,part=1
\begin{ocaml}
let _ = "Include me"
\end{ocaml}
